components/navbar/Navbar.js

```diff
@@ -124,7 +143,7 @@ export default function Navbar() {
           </div>
         </div>
 
-        <div className={`${!isOpen && "hidden"} md:hidden`} id="mobile-menu">
+        <div className={`${!isOpen && "hidden"} md:hidden`} id="mobile-menu" onClick={() => setIsOpen(false)}>
```

Comment:
currently clicking here will also close it, you can move the onClick function to the inner div 
![image](https://user-images.githubusercontent.com/95094057/211933787-a18cde05-8c40-42a9-86d1-6ad433c8a8a6.png)

Reply:
I am sorry for that.It's my bad. Thanks for spotting it. I will be making the following changes.